aws/v19.0.0-beta1/README.md

```diff
@@ -0,0 +1,166 @@
+
+:rotating_light: ***Cilium breaking changes***
+- The AWS CNI pod subnets are no longer used by Cilium. Please add custom routes with the node subnet(s) CIDR(s) instead of the AWS CNI pod subnets CIDR before upgrading to this release.
+- [Network Policy](https://docs.cilium.io/en/stable/concepts/kubernetes/policy/#networkpolicy-state) provided by Cilium does not cover support for setting [ipBlock with Pod IP](https://github.com/cilium/cilium/issues/9209). Components in need of this will have to use [CiliumNetworkPolicy](https://docs.cilium.io/en/stable/concepts/kubernetes/policy/#ciliumnetworkpolicy) which has wider scope. Please inspect your workloads and configured Network Policies carefully.
+- The upgrade process [documentation](https://handbook.giantswarm.io/docs/support-and-ops/ops-recipes/upgrade-to-cilium/) is available. Please read it carefully and reach out in case of questions.
+- Due to changes to CRs during upgrade the `gitops` automation will have to be suspended and any applied changes backported to the repos before resuming.
+
+:rotating_light: ***IRSA breaking changes***
+- Please read the [updated documentation](https://docs.giantswarm.io/advanced/iam-roles-for-service-accounts/)
+- Please remember that `kiam-app` will be removed by default during the upgrade.
+- Please remember to adapt the IAM policies prior to upgrade as specified in the [documentation](https://docs.giantswarm.io/advanced/iam-roles-for-service-accounts/)
+- There have been breaking changes introduced to the `Cloudfront domain alias` to provide greater predictability and enable easier automation of AWS IAM role creation by customers. Please adjust AWS IAM roles accordingly to the [documentation](https://docs.giantswarm.io/advanced/iam-roles-for-service-accounts/#aws-release-v19) prior to the upgrade.
+
+***General highlights***
+- The [k8s-dns-node-cache-app](https://github.com/giantswarm/k8s-dns-node-cache-app/) is now deployed by default. After the upgrade please delete the application if you have deployed it through managed catalog. Afterwards the app will be installed automatically.
+- The [prometheus-blackbox-exporter](https://github.com/giantswarm/prometheus-blackbox-exporter/) is a new monitoring component deployed by default. This is a blackbox monitoring tool that will validate internal, DNS and external connectivity.
+- `Cilium` will have Hubble UI enabled by default for troubleshooting and observability.
+
+## Change details
+
+
+### app-operator [6.6.3](https://github.com/giantswarm/app-operator/releases/tag/v6.6.3)
+
+### Changed
+
+- Lowered resource requests and limits
+- Changed VPA to consider unique and workload cluster operators as well and added support for min allowed fields of CPU and memory
+
+
+### aws-operator [14.12.0](https://github.com/giantswarm/aws-operator/releases/tag/v14.12.0)
```

Comment:
This is the same aws-operator as v18.3.0, this version should be bumped

Reply:
https://github.com/giantswarm/releases/commit/8ae82a7cdc441076e3444c9b7250d868b4409519#diff-6e52ea47f3498bb65265b38b0a56c250476e1187901593c62a34d0d2ac6c57ceR61